hugo/assets/scripts/arithmetics.tsx

```diff
@@ -0,0 +1,99 @@
+import { MonacoEditorReactComp, monaco, addMonacoStyles } from '@typefox/monaco-editor-react/bundle';
+import { buildWorkerDefinition } from 'monaco-editor-workers';
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+
+buildWorkerDefinition(
+  "../../libs/monaco-editor-workers/workers",
+  new URL("", window.location.href).href,
+  false
+);
+addMonacoStyles('monaco-editor-styles');
+ 
+const syntaxHighlighting = {
+    keywords: [
```

Comment:
Move that constants to an own file, then the reader can focus better on the rest. Keep in mind that seeing such a big thing requires the reader to switch his mind to a different syntax/concept, which can be disturbing